lib/Elastica/Index.php

```diff
@@ -682,24 +642,19 @@ public function setSettings(array $data)
      *
      * @param string       $path   Path to call
      * @param string       $method Rest method to use (GET, POST, DELETE, PUT)
-     * @param array|string $data   OPTIONAL Arguments as array or encoded string
-     * @param array        $query  OPTIONAL Query params
-     *
-     * @return Response Response object
+     * @param array|string $data   Arguments as array or encoded string
      */
-    public function request($path, $method, $data = [], array $query = [])
+    public function request($path, string $method, $data = [], array $queryParameters = []): Response
```

Comment:
I guess the first param is also a string?

Reply:
Fixed!